docs/TEAM.md

```diff
@@ -72,8 +72,13 @@
 | <img src="./images/team/marcus.jpg" width="80px">                                                                                                                                                                                                              | [Marcus](https://github.com/mar-code)                     | I'm a self taught frontend developer from Lagos, Nigeria with experience in HTML/CSS, Javascript, React, Redux, bootstrap and d3 data presentation. i am currently learning NodeJs.                                                                                                                                         |
 |                                                                                                                                                                                                                                                                | [Samuel Adeniyi](https://github.com/samueladeniyi)        | I am proficient in HTML, CSS and Javascript. I am comfortable with react. I intend to become a fullstack developer with the MERN stack. I am from Nigeria.                                                                                                                                                                  |
-| <img src="./images/team/aditya.jpg" width="80px">                                                                                                                                                                                                              | [Aditya Satpute](https://github.com/Foxtrot-1418)                     | I'm a Full Stack Web Developer From Nashik, Maharashtra, India. I've experience in HTML, CSS, JS, Jquery, Bootstrap, PHP and currently working fo r Multinational Company as AWS Developer.                                                                                                                   |
- |
+| <img src="./images/team/aditya.jpg" width="80px">                                                                                                                                                                                                              | [Aditya Satpute](https://github.com/Foxtrot-1418)                     | I'm a Full Stack Web Developer From Nashik, Maharashtra, India. I've experience in HTML, CSS, JS, Jquery, Bootstrap, PHP and currently working fo r Multinational Company as AWS Developer.                                                                                                                                                                |
+ | <img src="./images/team/simrin_joshi.jpg" 
+ width="80px">      
```

Comment:
Keep the <img> code in the same line.

![image](https://user-images.githubusercontent.com/46648727/101283375-a8c62480-37b8-11eb-9aa3-c6d2c40c416b.png)